openapi-cli/src/main/java/io/ballerina/openapi/generators/schema/TypeGeneratorUtils.java

```diff
@@ -155,17 +161,36 @@ public static TypeDescriptorNode getNullableType(Schema schema, TypeDescriptorNo
             MetadataNode metadataNode = createMetadataNode(documentationNode, createEmptyNodeList());
             if (required != null) {
                 if (!required.contains(field.getKey().trim())) {
-                    recordFieldNode = NodeFactory.createRecordFieldNode(metadataNode, null,
-                            fieldTypeName, fieldName, createToken(QUESTION_MARK_TOKEN), createToken(SEMICOLON_TOKEN));
+                    if (field.getValue().getDefault() != null) {
```

Comment:
could you please let me know if we have an empty string as default value ` ex: url:" "`, what would be our output, are we going to add an empty string there? 🙂